explore/src/main/scala/explore/tabs/SiderealTargetEditorTile.scala

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.tabs
+
+import cats.effect.IO
+import cats.syntax.all._
+import crystal.react.View
+import crystal.react.implicits._
+import crystal.react.reuse._
+import explore.components.Tile
+import explore.components.ui.ExploreStyles
+import explore.model.TargetVisualOptions
+import explore.model.reusability._
+import explore.targeteditor.SiderealTargetEditor
+import explore.undo.UndoStacks
+import japgolly.scalajs.react.vdom.html_<^._
+import lucuma.core.model.Target
+import lucuma.core.model.User
+import lucuma.ui.reusability._
+import react.common._
+
+object SiderealTargetEditorTile {
+
+  def sideralTargetEditorTile(
```

Comment:
Where do we use this one?

Reply:
When editing an individual target. ie. not editing an asterism.